.eslintrc.cjs

```diff
@@ -148,7 +148,7 @@ module.exports = defineConfig({
       }
     },
     {
-      files: ['playground/**'],
+      files: ['playground/**', '**/__tests__/**'],
```

Comment:
There's a lint error that I can't use `new URL` in `__tests__`, so I added this here to get the other special lint rules for testing.